code/modules/admin/verbs/adminhelp.dm

```diff
@@ -79,23 +79,30 @@
 	if(src.handle_spam_prevention(msg,MUTE_ADMINHELP))
 		return
 
+	for(var/datum/adminticket/T in admintickets)
+		if(T.permckey == src.ckey && T.resolved == "No")
```

Comment:
Can users have more than one active ticket? If so won't this repeat for each one?